### PR TITLE
AF-546: Reduce startup time of AppFormer apps and workbenches

### PR DESCRIPTION
**Pull request by mbarkley**
* Replace @AfterInitialization methods with @PostConstruct so that
startup RPC can be grouped into a single payload.
* Make some slow @EntryPoint beans into @ApplicationScoped so that
they do not need to slow down startup
* Make HomeModel proxiable (for change in kie-drools-wb-webapp entrypoint)
* Removed PlaceManagerActivityService

### CODE REVIEW

**Comment by mbarkley** [comment]
Relates to:
* https://github.com/errai/errai/pull/267
* https://github.com/AppFormer/uberfire/pull/751
* https://github.com/dashbuilder/dashbuilder/pull/340
* https://github.com/kiegroup/guvnor/pull/464
* https://github.com/kiegroup/drools-wb/pull/533
* https://github.com/kiegroup/jbpm-form-modeler/pull/117
* https://github.com/kiegroup/kie-wb-distributions/pull/542
* https://github.com/kiegroup/optaplanner-wb/pull/181
* https://github.com/kiegroup/appformer/pull/88
* https://github.com/kiegroup/jbpm-wb/pull/764

File: kie-wb-common-services/kie-wb-common-services-api/src/main/java/org/kie/workbench/common/services/shared/service/PlaceManagerActivityService.java [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
- */
-
-package org.kie.workbench.common.services.shared.service;
-
-import java.util.List;
-
-import org.jboss.errai.bus.server.annotations.Remote;
-
-@Remote
-public interface PlaceManagerActivityService {
```
Comment (manstis) [inline kie-wb-common-services/kie-wb-common-services-api/src/main/java/org/kie/workbench/common/services/shared/service/PlaceManagerActivityService.java]: @mbarkley @paulovmr What did ```PlaceManagerActivityService``` actually do? It looks like it held a server-side list of client-side ```ActivityId```s.... what does it's removal mean?

File: kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-api/src/main/java/org/kie/workbench/common/screens/home/model/HomeModel.java [hunk 2]
```diff
@@ -30,6 +30,11 @@
     private final List<CarouselEntry> carouselEntries = new ArrayList<>();
     private final List<SectionEntry> sections = new ArrayList<>();
 
+    // For proxying
+    protected HomeModel() {
+      this.title = null;
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-api/src/main/java/org/kie/workbench/common/screens/home/model/HomeModel.java]: Can you please format this correctly (4 spaces).

File: kie-wb-common-services/kie-wb-common-services-api/src/main/java/org/kie/workbench/common/services/shared/service/PlaceManagerActivityService.java (hunk 1 above)
Comment (paulovmr) [inline kie-wb-common-services/kie-wb-common-services-api/src/main/java/org/kie/workbench/common/services/shared/service/PlaceManagerActivityService.java]: @manstis It was only used [here](https://github.com/kiegroup/jbpm-wb/blob/6.5.x/jbpm-console-ng-human-tasks-forms/jbpm-console-ng-human-tasks-forms-backend/src/main/java/org/jbpm/console/ng/ht/forms/backend/server/PlaceManagerFormProvider.java#L44) on jbpm-wb until 6.5.x, but in 7.0.x it stop being used afaik.

Comment (manstis) [inline kie-wb-common-services/kie-wb-common-services-api/src/main/java/org/kie/workbench/common/services/shared/service/PlaceManagerActivityService.java]: Thanks @paulovmr

**Comment by mbarkley** [comment]
@manstis Fixed the formatting mistake.

**Pull request merged**